components/dateinput/overview.md

```diff
@@ -10,119 +10,144 @@ position: 0
 
 # Blazor DateInput Overview
 
-The <a href="https://www.telerik.com/blazor-ui/date-input" target="_blank">Blazor Date Input component</a> allows the user to enter a date. The developer can control the format of the date. If the user input does not match the desired pattern, the value is not accepted. If the input can be parsed, it will be corrected automatically.
+The <a href="https://www.telerik.com/blazor-ui/date-input" target="_blank">Blazor Date Input component</a> allows the user to type a date in a more convenient and user-friendly way, compared to a regular textbox. The DateInput can display its value with a specific date format and hin the user to follow it during typing. The component also provides multiple settings that are related to the typing and auto-correction user experience. The DateInput is a base for other components such as the [DatePicker]({%slug components/datepicker/overview%}), [DateTimePicker]({%slug components/datetimepicker/overview%}) and [DateRangePicker]({%slug daterangepicker-overview%}).
+
 
 ## Creating Blazor DateInput
 
-1. Add the `TelerikDateInput` tag to your razor page.
-1. Bind a `DateTime` object to the component
-1. Optionally, provide custom `Format`, `Min` and `Max` values
+1. Add a `TelerikDateInput` tag to your razor page.
+1. Bind the `Value` parameter to a [`DateTime` or `DateTime?` object](#nullable-datetime). The parameter supports two-way binding.
+1. (optional) Set the [`Format`, `Min` and `Max` parameters](#date-input-parameters).
+1. (optional) Configure the [typing user experience](#typing-settings) related to automatic value correction or navigation across the different parts of the date format.
 
->caption Basic DateInput with custom format, min and max
+>caption Basic Telerik Blazor DateInput
 
 ````CSHTML
-@dateInputValue
-<br />
-
-<TelerikDateInput @bind-Value="@dateInputValue" Format="dd MMMM yyyy"
-                  Min="@Min" Max="@Max">
+<TelerikDateInput @bind-Value="@DateValue"
```

Comment:
The sample does not include a demonstration of the 4th bullet. Maybe we can enable some of the new parameters.